handlers.go

```diff
@@ -432,6 +443,130 @@ func (s *Server) handleStats(c echo.Context, u *User) error {
 	return c.JSON(200, out)
 }
 
+// handlePeeringPeersAdd godoc
+// @Summary      Add peers on Peering Service
+// @Description  This endpoint can be used to add a Peer from the Peering Service
+func (s *Server) handlePeeringPeersAdd(c echo.Context) error {
+	var params []peering.PeeringPeer
+	if err := c.Bind(&params); err != nil {
+		return err
+	}
+
+	for _, peerParam := range params {
+
+		var multiAddrs []multiaddr.Multiaddr
+		for _, addr := range peerParam.Addrs {
+			a, err := multiaddr.NewMultiaddr(addr)
+			if err != nil {
+				return &util.HttpError{
+					Code:    400,
+					Message: util.ERR_PEERING_PEERS_ADD_ERROR,
+				}
+			}
+			multiAddrs = append(multiAddrs, a)
+		}
+
+		peerParamId, _ := peer.Decode(peerParam.ID)
+		//	add the peer(s)
+		s.Node.Peering.AddPeer(
+			peer.AddrInfo{
+				peerParamId,
+				multiAddrs,
+			})
+	}
+	return c.JSON(200, params)
+}
+
+// handlePeeringPeersRemove godoc
+// @Summary      Remove peers on Peering Service
+// @Description  This endpoint can be used to remove a Peer from the Peering Service
+func (s *Server) handlePeeringPeersRemove(c echo.Context) error {
+	var params []peer.ID
+
+	type PeeringPeerRemoveMessage struct {
+		Message     string    `json: Message`
+		PeersRemove []peer.ID `json: Peers`
+	}
+
+	if err := c.Bind(&params); err != nil {
+		return &util.HttpError{
+			Code:    400,
+			Message: util.ERR_PEERING_PEERS_REMOVE_ERROR,
+		}
+	}
+
+	for _, peerId := range params {
+		s.Node.Peering.RemovePeer(peerId)
+	}
+
+	return c.JSON(200, PeeringPeerRemoveMessage{"Removed the following Peers from Peering", params})
+}
+
+// handlePeeringPeersList godoc
+// @Summary      List all Peering peers
+// @Description  This endpoint can be used to list all peers on Peering Service
+func (s *Server) handlePeeringPeersList(c echo.Context) error {
+	var connectionCheck []peering.PeeringPeer
+	for _, peerAddrInfo := range s.Node.Peering.ListPeers() {
+
+		var peerAddrInfoAddrsStr []string
+		for _, addrInfo := range peerAddrInfo.Addrs {
+			peerAddrInfoAddrsStr = append(peerAddrInfoAddrsStr, addrInfo.String())
+		}
+		connectionCheck = append(connectionCheck, peering.PeeringPeer{
+			ID:        peerAddrInfo.ID.Pretty(),
+			Addrs:     peerAddrInfoAddrsStr,
+			Connected: (s.Node.Host.Network().Connectedness(peerAddrInfo.ID) == network.Connected),
+		})
+	}
+	return c.JSON(200, connectionCheck)
+}
+
+// handlePeeringStart godoc
+// @Summary      Start Peering
+// @Description  This endpoint can be used to start the Peering Service
+func (s *Server) handlePeeringStart(c echo.Context) error {
+	type GenericResponse struct {
+		Message string `json: "Message"`
+	}
```

Comment:
This is declared twice (here and on `handlePeeringStop`). Maybe leave it on the top of the file

Reply:
yeah initially I was thinking of creating 2 separate models for these 2 types of response but I think it make sense to just use a generic one. I'll put this at the top then. Thank you.